src/ResourceManager/IotHub/Commands.IotHub/Models/PSCertificatePropertiesWithNonce.cs

```diff
@@ -0,0 +1,75 @@
+﻿namespace Commands.IotHub.Models
+{
+    class PSCertificatePropertiesWithNonce
```

Comment:
delete everything above the copyright message.

Reply:
Deleted